buildSrc/src/main/kotlin/scenery/base.gradle.kts

```diff
@@ -102,7 +102,7 @@ tasks {
     register<JavaExec>("compileShader") {
         group = "tools"
         mainClass.set("graphics.scenery.backends.ShaderCompiler")
-        classpath = sourceSets["main"].runtimeClasspath
+        classpath = sourceSets["test"].runtimeClasspath
```

Comment:
I dont understand this. Do we need shaders only when testing?

Reply:
No, what this block of code does is to register a Gradle task with the name `compileShader`. The change from the main to the test classpath only means that some libraries that are usually only in the test classpath will be available (such as picocli or slf4j).

Reply:
I actually thought some more about this and found a better solution now. The classpath I've also reverted to `main`, which makes more sense.